api/clients/v2/payloadretrieval/validator_payload_retriever.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/Layr-Labs/eigenda/api/clients/v2/coretypes"
 	"github.com/Layr-Labs/eigenda/api/clients/v2/validator"
 	"github.com/Layr-Labs/eigenda/api/clients/v2/verification"
+	core "github.com/Layr-Labs/eigenda/core/v2"
```

Comment:
You're importing `"github.com/Layr-Labs/eigenda/core/v2"` twice, under two different aliases.

Curious that go with all of its opinionated import rules doesn't complain about this

Reply:
I'm also surprised actually. I think go vet or some linter (revive?) will catch this though. Which I added as part of https://github.com/Layr-Labs/eigenda/pull/1604 (revive is not turned on though.. because it was complaining about too many things that I didn't feel like fixing lol)

Reply:
[97e1641](https://github.com/Layr-Labs/eigenda/pull/1607/commits/97e1641e24de5dbfeb50f7ff41cb6e23529f4ab9)